templates/core/index.html

```diff
@@ -63,7 +63,7 @@ <h2>An OS with all the infrastructure</h2>
         production-ready images, unlock over-the air (OTA) updates, deploy in air gap environments, manage your fleet of
         devices, achieve real-time operations, and more.
       </p>
-      <p>Ubuntu Core will transform the way you deploy devices with the longest support window in the industry.</p>
+      <p>Ubuntu Core will transform the way you deploy devices with the longest support window in the industry&mdash;12 years.</p>
```

Comment:
```suggestion
      <p>Ubuntu Core will transform the way you deploy devices with the longest support window in the industry &mdash; 12 years.</p>
```

Reply:
I think there is [no space in em dash](https://academicguides.waldenu.edu/writingcenter/punctuation/dashes#:~:text=For%20both%20en%20dashes%20and%20em%20dashes%2C%20do%20not%20put%20any%20spaces%20before%20or%20after%20the%20dashes.).

Reply:
I don't know about mdash but contextually I think there should be a space. Otherwise it looks like one word "industry-12"